app/views/error.njk

```diff
@@ -5,5 +5,6 @@
 {% block contentBody %}
   <h1 class="govuk-heading-l">{{ __p('error.title') }}</h1>
 
+  {% if messageAmountReference %}<div id="errorMsg" class="govuk-body" data-cy="error-message">{{ __p(messageAmountReference) }}</div>{% endif %}
```

Comment:
I’m not sure about the name `messageAmountReference` — it’s very specific when it seems like what we’re doing in the context of this template is adding support for an additional paragraph to be included in the error message. Perhaps `messagePreamble` would be better.

In addition, the ID `errorMsg` duplicates the ID used below.

We also have two consecutive `div` elements rather than `p` elements but when I look into it there’s actually quite a few ways in which this page does not conform to the standard template for error messages, so I’ll create another ticket for fixing that and we can address the double `div`s then.

Reply:
Variable changes as suggested.